src/app/templates/index.html

```diff
@@ -56,5 +56,62 @@ <h2>Usage</h2>
         </li>
       </ul>
     </div>
+    <div style="width: 500px; margin: 0 auto">
+      <h2>Add New Feed</h2>
+      <form method="post" action="{{ url_for('main.add_feed') }}">
+        <label for="feed_url">Feed URL:</label>
+        <input type="text" id="feed_url" name="url" required />
+        <button type="submit">Add Feed</button>
```

Comment:
Is the plan to eventually deprecate the feeds section of the config in favor of the DB version?

Reply:
yeah, that makes sense. It'll simplify things when the processing portion interacts with the DB